test/MorphoTokenEthereumTest.sol

```diff
@@ -237,6 +239,7 @@ contract MorphoTokenEthereumTest is BaseTest {
 
     function testMint(address to, uint256 amount) public {
         vm.assume(to != address(0));
+        vm.assume(to != address(wrapper));
```

Comment:
We could remove this and make the check on balance more accurate instead (being equal to amount + amountBefore)